conf/messages

```diff
@@ -159,13 +159,14 @@ setupIncomplete.p1=<a href="{0}">Try again using a different agent user ID</a> i
 invasive.title.saCode=Do you have a Self Assessment agent code?
 invasive.title.option=Add some detail about a taxpayer related to the SA agent code you entered
 invasive.p1=You must choose from these 2 options so we can identify one of your current clients.
-invasive.confirmResponse.true.yes=Yes
-confirmResponse.true.hidden.input.label.sa=Self Assessment agent code
-invasive.confirmResponse.false.no=No
-invasive.confirmResponse.true.nino=Taxpayer''s National Insurance (NI) number
-confirmResponse.true.hidden.input.label.nino=NI number
-invasive.confirmResponse.true.utr=Taxpayer''s Self Assessment Unique Taxpayer Reference (UTR)
-confirmResponse.false.hidden.input.label.utr=Self Assessment UTR
+invasive.radioResponse.yes=Yes
+radioResponse.hidden.input.label.sa=Self Assessment agent code
```

Comment:
Perhaps add `invasive.` at the start of this message key to keep it consistent with the others? Likewise with lines 166, and 168